src/Illuminate/Redis/Connections/PhpRedisClusterConnection.php

```diff
@@ -4,5 +4,20 @@
 
 class PhpRedisClusterConnection extends PhpRedisConnection
 {
-    //
+    /**
+     * Flush the selected Redis database.
+     *
+     * @param  string|null  $modifier
+     * @return mixed
+     */
+    public function flushdb($modifier = null)
+    {
+        $async = strtoupper($modifier) === 'ASYNC';
```

Comment:
You may want to `strtoupper($modifier ?? '')` for PHP 8.1+ because standard library string functions no longer allowing `null`  arguments was a feature devs have been absolutely clamoring for.

```php
strtoupper(null);
// PHP Deprecated:  strtoupper(): Passing null to parameter #1 ($string) of type
// string is deprecated in eval()'d code on line 1
```